bench/vector_tile.js

```diff
@@ -56,9 +56,9 @@ Tile.Feature.read = function (pbf, end) {
 };
 Tile.Feature._readField = function (tag, obj, pbf) {
     if (tag === 1) obj.id = pbf.readVarint();
-    else if (tag === 2) obj.tags = pbf.readPackedVarint();
+    else if (tag === 2) obj.tags = pbf.readPackedVarint(obj.tags);
```

Comment:
It's only necessary if the library should be forwards compatible (e.g., new compiled proto with old `pbf`). If that's not a concern, no problem, it saves a few ops. :smile: